src/Components/WebAssembly/Authentication.Msal/src/Microsoft.Authentication.WebAssembly.Msal.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
-  <Sdk Name="Yarn.MSBuild" />
+  <Sdk Name="Yarn.MSBuild" Condition=" '$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
nit: Might want to add an `<Error />` in one of the `$(ResolveStaticWebAssetsInputsDependsOn)` targets if `'$(DotNetBuildFromSource)' == 'true'` because the failure to find `yarn` or Yarn.MSBuild will likely be slightly obscure. Same in src/Components/WebAssembly/WebAssembly.Authentication/src/Microsoft.AspNetCore.Components.WebAssembly.Authentication.csproj

Reply:
You mean in case we ever turn these on for SourceBuild?

Reply:
Yeah, fast failure sounds good for that eventuality

Reply:
Updated